### PR TITLE
Prepare to use tslint

### LINKED ISSUE #22: Using tslint or any other linter
```sh
 $ tslint --init
 $ tslint -c tslint.json '{.,test}/src/**/*.ts' | wc -l
2281
```
Most of these two thousand are suggestions of missing semicolons or using constant when a variable was never reassigned. Most of these can be fixed automatically by `tslint`. What do you think about conforming to these suggestions and checking it by CI before merging?

### PR DESCRIPTION
**Pull request by przepompownia**
Closes #22

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,5 +41,8 @@
     "vscode-uri": "^1.0.6",
     "webpack": "^4.41.2",
     "webpack-cli": "^3.3.9"
+  },
+  "dependencies": {
+    "tslint": "^5.20.0"
```
Comment (iamcco) [inline package.json]: tslint should be dev dependencies

Comment (przepompownia) [reply package.json]: Right, I forgotten add a parameter to a command.

**Comment by przepompownia** [comment]
Some from the rest reported errors (like shadowed variable names) may require to take a while and more attention.

**Comment by iamcco** [comment]
Thanks for your work!

**Comment by przepompownia** [comment]
Thanks also. It was incomplete but the rest can be done in any separate PR.

**Comment by iamcco** [comment]
> It was incomplete but the rest can be done in any separate PR.

Yes, this PR is become too big, i think merge this first will make sense

**Pull request merged**